allure-junit5/src/test/java/io/qameta/allure/examples/junit5/AllureParameterizedTest.java

```diff
@@ -12,13 +12,11 @@
 public class AllureParameterizedTest {
 
     @ParameterizedTest(name = "{displayName} [{argumentsWithNames}]")
-    @ValueSource(strings = {"First Name", "Second Name"})
+    @ValueSource(strings = {"John", "Mike"})
     @DisplayName("allureParameterizedTest displayName")
     @Description("allureParameterizedTest description")
-    public void allureParameterizedTest(String testParam) {
-        parameter("testParam", testParam);
-        step("Step inside parameterized test");
-        step("Test parameter: " + testParam);
+    public void allureParameterizedTest(String name) {
+        parameter("Name", name);
     }
 
     @Test
```

Comment:
do we really need the allureFakeParameterizedTest below?

The allureParameterizedTest above already contains the example of "dynamic usage" of parameter method. Isn't it enough? Which other purpose does the allureFakeParameterizedTest method reflect in this test class?

But ok... Maybe the main purpose is to show "dynamic call of parameter" method... Then another option would be – keep allureFakeParameterizedTest with `parameter(...)` inside, but remove it from `allureParameterizedTest`. In this way we would decompose the complicated `allureParameterizedTest` test into simpler smaller examples. From other point of view - the `allureParametrizedTest` example being not simple in implementation (a lot of annotations used) – looks pretty natural in context of details reflected:

<img width="709" alt="image" src="https://user-images.githubusercontent.com/1961118/112057119-8be41000-8b61-11eb-8144-adf3b04dbcd9.png">

Maybe I would just remove `allureFakeParameterizedTest` then... but not sure:)